ubiquibot-config-default.json

```diff
@@ -93,37 +93,12 @@
       "enabled": false
     }
   ],
-  "comment-element-pricing": {
-    "blockquote": 0,
-    "break": 0,
-    "code": 0,
-    "definition": 0,
-    "emphasis": 0,
-    "heading": 0,
-    "html": 0,
-    "image": 0,
-    "imageReference": 0,
-    "inlineCode": 0,
-    "link": 0,
-    "linkReference": 0,
-    "list": 0,
-    "listItem": 0,
-    "paragraph": 0,
-    "root": 0,
-    "strong": 0,
-    "text": 0,
-    "thematicBreak": 0,
-    "delete": 0,
-    "footnoteDefinition": 0,
-    "footnoteReference": 0,
-    "table": 0,
-    "tableCell": 0,
-    "tableRow": 0,
-    "alignType": 0,
-    "flowContent": 0,
-    "listContent": 0,
-    "phrasingContent": 0,
-    "rowContent": 0,
-    "tableContent": 0
```

Comment:
These should all be initialized to `0` why was this removed?

Reply:
Because they every element is 0 by default

Reply:
This is the `default` configuration so I thought it makes more sense to explicitly set defaults here instead of inferring them inside of the loader?

Reply:
I agree that the default incentives config should be explicit because right now it is not clear what parameters are available

This [one](https://github.com/inikulin/parse5/blob/7f8c43c957ea60461d3afd55ca1905c095452f40/packages/parse5/lib/common/html.ts#L33) seems to be a comprehensive list of all available tags (supported by [parse5](https://parse5.js.org/index.html))

Adding all of the tags to the config is redundant but we can add at least most commonly used like `h1`, `img`, `label`, etc...

@whilefoo What do you think?

Reply:
Ok I agree, we can have the most common tags so it will be easier for partners to understand

Reply:
> Ok I agree, we can have the most common tags so it will be easier for partners to understand. Shall I create a PR?

This would be awesome, thank you

Reply:
> I agree that the default incentives config should be explicit because right now it is not clear what parameters are available
> 
> 
> 
> This [one](https://github.com/inikulin/parse5/blob/7f8c43c957ea60461d3afd55ca1905c095452f40/packages/parse5/lib/common/html.ts#L33) seems to be a comprehensive list of all available tags (supported by [parse5](https://parse5.js.org/index.html))
> 
> 
> 
> Adding all of the tags to the config is redundant but we can add at least most commonly used like `h1`, `img`, `label`, etc...
> 
> 
> 
> @whilefoo What do you think?

Wait I already fixed this in my pull request. I need to find it which is tedious from my phone, but if you read this comment before I find it, please check the open pull requests from four days ago. Check my newest one that is not a draft.